samples/apps/copilot-chat-app/webapp/src/redux/features/message-relay/signalRMiddleware.ts

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import * as signalR from "@microsoft/signalr";
+import { AlertType } from "../../../libs/models/AlertType";
+import { IChatUser } from "../../../libs/models/ChatUser";
+import { IAskResult } from "../../../libs/semantic-kernel/model/AskResult";
+import { addAlert } from "../app/appSlice";
+import { AuthorRoles, ChatMessageState, IChatMessage } from './../../../libs/models/ChatMessage';
+import { isPlan } from './../../../libs/utils/PlanUtils';
+import { getSelectedChatID } from './../../app/store';
+import { FileUploadedAlert } from './../conversations/ChatState';
+
+// These have to match the callback names used in the backend
+const receiveMessageFromServerCallbackName = "ReceiveMessage" as string;
```

Comment:
I think this would make more sense as an enum rather than separate constant strings

i.e., 
```
const enum ServerCallbackNames {
     ReceiveMessage = 'RecieveMesage'
    ....
}
```